// @open-web3/orml-types/interfaces/definitions.js
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "runtime", {
  enumerable: true,
  get: function () {
    return _definitions.default;
  }
});
Object.defineProperty(exports, "authority", {
  enumerable: true,
  get: function () {
    return _definitions2.default;
  }
});
Object.defineProperty(exports, "oracle", {
  enumerable: true,
  get: function () {
    return _definitions3.default;
  }
});
Object.defineProperty(exports, "tokens", {
  enumerable: true,
  get: function () {
    return _definitions4.default;
  }
});
Object.defineProperty(exports, "traits", {
  enumerable: true,
  get: function () {
    return _definitions5.default;
  }
});
Object.defineProperty(exports, "vesting", {
  enumerable: true,
  get: function () {
    return _definitions6.default;
  }
});
Object.defineProperty(exports, "graduallyUpdates", {
  enumerable: true,
  get: function () {
    return _definitions7.default;
  }
});
Object.defineProperty(exports, "rewards", {
  enumerable: true,
  get: function () {
    return _definitions8.default;
  }
});

var _definitions = _interopRequireDefault(require("./runtime/definitions"));

var _definitions2 = _interopRequireDefault(require("./authority/definitions"));

var _definitions3 = _interopRequireDefault(require("./oracle/definitions"));

var _definitions4 = _interopRequireDefault(require("./tokens/definitions"));

var _definitions5 = _interopRequireDefault(require("./traits/definitions"));

var _definitions6 = _interopRequireDefault(require("./vesting/definitions"));

var _definitions7 = _interopRequireDefault(require("./graduallyUpdates/definitions"));

var _definitions8 = _interopRequireDefault(require("./rewards/definitions"));
